### PR TITLE
tools: Reorganize block generator scripts.

### PR DESCRIPTION
**Pull request by winder**
## Summary

Cleanup the block-generator root directory by moving dev scripts into a `script` subdirectory and removing the `run_test.sh` script.

### CODE REVIEW

File: tools/block-generator/scripts/run_runner.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def parse_args():
     parser.add_argument("--conduit-binary", help="Path to conduit binary")
     parser.add_argument(
         "--scenario",
-        default=(default := CWD / "test_config.yml"),
+        default=(default := CWD.parents[1] / "test_scenario.yml"),
```
Comment (winder) [inline tools/block-generator/scripts/run_runner.py:85]: @tzaffi fyi, you might need to fix this next time you're working on block generator.

**Review by tzaffi — COMMENTED** [review]
This all looks reasonable. I think it's worth seeing which commands in the `Makefile` have been broken -if any- and then for the broken ones, triage and decide whether to:
* fix
* remove
* comment out with a TODO comment indicating future fix

**Review by shiqizng — APPROVED** [review]

**Comment by winder** [comment]
@tzaffi Thanks, I think I fixed the only Makefile problem.

**Review by shiqizng — APPROVED** [review]

**Review by tzaffi — APPROVED** [review]
LGTM

**Pull request merged**